### ISSUE TITLE
aspnet beta6 The dependency Serilog >= 1.5.9 could not be resolved

### ISSUE THREAD

**Issue by rswillis** [serilog/serilog#503]
Hi, I've been using versions of Serilog rolling file logging with prior aspnet betas and hadn't given it much thought.  When beta6 was released, however, I began getting a hard dependency resolution error.  I suspect it has to do w. the updated version of the System.Runtime that EF7 beta6 is wanting.  Can anyone confirm, and are there plans to address?  Serilog has been super helpful troubleshooting.  I do realize this pertains to a beta release, so apologies if my question is misplaced or ill timed.  Just wanted to make sure I wasn't doing something wrong.  

thx & rgds, Randall

{
  "version": "1.0.0-*",
  "description": "myApp.Data Class Library",
  "authors": [ "Randall" ],
  "tags": [ "" ],
  "projectUrl": "",
  "licenseUrl": "",

  "dependencies": {
    "System.Collections": "4.0.10-beta-23109",
    "System.Linq": "4.0.0-beta-23109",
    "System.Threading": "4.0.10-beta-23109",
    "System.Runtime": "4.0.20-beta-23109",
    "Microsoft.CSharp": "4.0.0-beta-23109",
    "disruptor.Definition": "1.0.0-*",
    "EntityFramework.SqlServer": "7.0.0-beta6",
    "EntityFramework.Commands": "7.0.0-beta6"
  },

  "frameworks": {
    "dnx451": {
      "dependencies": {
"Serilog": "1.5.9"

      }
    },
    "dnxcore50": { }
  }
}

**Comment by nblumhardt** [comment]
Hi Randall,

Thanks for the heads-up. I'm not aware of any specific issue here, so looks like you're trailblazing :-)

If it helps, you might find some clues over at https://github.com/serilog/serilog-framework-logging

HTH,
Nick

**Comment by rswillis** [comment]
Thanks for the reply.  I sat down this evening to distill this to some reproducible steps,  I added a fresh web (api) project, added serilog, no problems.  I added a portable class library, and added EF7 as before, only this time I didn't even get to add serilog before EF7 threw this:  

The dependency EntityFramework.SqlServer 7.0.0-beta6 does not support framework .NETPlatform,Version=v5.0.

I can see a path forward by simply pulling EF7/serilog/and my code all back into the main web project for now and letting the beta6 dependency stew cook a bit longer before spinning off assemblies.  onward...

closing this issue.  cheers, Randall
